### PR TITLE
Add config for 'No team'

### PR DESCRIPTION
**Pull request by lucasmrod**
Just adding the "No team" file.

### CODE REVIEW

File: default.yml [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
+# For Fleet Free, this file updates policies, queries, and agent_options for all hosts.
 
-# For Fleet Premium, the default.yml file updates policies and queries that run on all hosts ("All teams"). Controls and agent options are updated for hosts on "No team."
+# For Fleet Premium, the default.yml file updates policies and queries that run on all hosts ("All teams"). 
 
 policies:
 queries:
 agent_options:
   path: ./lib/agent-options.yml
-controls:
```
Comment (noahtalerman) [inline default.yml:9]: If we remove controls, how will the Fleet Free user add configuration profiles?

Maybe we instruction Fleet Premium users only to remove `controls`?

**Review by noahtalerman — DISMISSED** [review]
Looking good! 

Yes, I think it makes sense to remove the software and policies. And put them in dogfood instead. We've learned that this gitops repo gets out of date fast. One more spot to maintain. 

We can always point folks to our dogfood `it-and-security/` folder. It's public!

File: default.yml (hunk 1 above)
Comment (lucasmrod) [reply default.yml:9]: Good point, I forgot about Fleet Free, we can leave it here empty.

File: teams/no-team.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Teams are available in Fleet Premium. 
+
+# This file updates policies, controls, and software for hosts assigned to "No team."
```
Comment (noahtalerman) [inline teams/no-team.yml:3]: @lucasmrod is it true that we only support policies, controls, and software for "No team"? (no queries or agent options yet)

And, queries and agent options for hosts on "No team" are controlled at the global (default.yml) level.

**Review by noahtalerman — COMMENTED** [review]
Hey @lucasmrod I took a pass that the comment section in the default.yml and teams/ YAML files to make them consistent and explicit.

When you get the chance, please take a look to see if they're accurate. Thanks!

File: teams/no-team.yml (hunk 2 above)
Comment (lucasmrod) [reply teams/no-team.yml:3]: - Queries: We don't support queries for "No team", so, queries defined in `default.yml` are global queries.
- Agent options: Agent options for "No team" hosts are defined in `default.yml` (in the future we could move this to `teams/no-team.yml`). On this iteration Marko adviced (in the Figma designs) to throw a warning if the user is setting `agent_options` or `queries` in `teams/no-team.yml` (they will be ignored).

**Comment by lucasmrod** [comment]
> When you get the chance, please take a look to see if they're accurate. Thanks!

@noahtalerman Changes look good!

File: teams/no-team.yml (hunk 2 above)
Comment (noahtalerman) [reply teams/no-team.yml:3]: Got it! Thanks :)

I updated the help text (comments at the top of each file) to be explicit. Please let me know if those doesn't look right.

**Review by noahtalerman — APPROVED** [review]

**Pull request merged**